tests/tests_basics_files.yml

```diff
@@ -305,7 +333,10 @@
       include_tasks: tasks/check_packages.yml
 
     - name: Check the filter
-      command: /bin/grep '\*.info;authpriv.none;auth.none;cron.none;mail.none.*{{ __default_system_log }}' '{{ __test_files_conf0 }}'
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space